src/client/js/scenes/SplashScene.js

```diff
@@ -1,31 +1,61 @@
 /* global Phaser */
+import Player from '../Player.js';
 
 export default class SplashScene extends Phaser.Scene {
   preload () {
-    this.load.setBaseURL('http://labs.phaser.io');
-
-    this.load.image('sky', 'assets/skies/space3.png');
-    this.load.image('logo', 'assets/sprites/phaser3-logo.png');
-    this.load.image('red', 'assets/particles/red.png');
+    // wtf i cant get this to load
+    this.load.image('goku', './assets/goku.png');
```

Comment:
I think this should be `this.add.image` Just looked at some of the phaser examples and it seems .load is for loading from an external url.